### ISSUE TITLE
Material Dark theme color issue with Visual Studio 2019

### ISSUE THREAD

**Issue by mikecoop** [Microsoft/VS-ColorThemes#64]
When using the Material Dark theme, the platforms under the different project types are unreadable.

![image](https://user-images.githubusercontent.com/1671683/53985592-057b4380-40ea-11e9-93f0-ec1046dc9b31.png)

**Comment by D36u92** [comment]
Yes, I can't find the report to change either. The SubText and the boxes. Where do I change them?

![fail theme](https://user-images.githubusercontent.com/36854869/54071450-e527c200-426c-11e9-98cb-a4c04fb38364.png)

**Comment by mikecoop** [comment]
I looked for quite a while but couldn't find the property that controls those boxes. I'm sure someone knows.

**Comment by D36u92** [comment]
I copied the default Black Theme from Windows. There all colors adjusted. It looks good like this. Don't find the property.

**Comment by mcquiggd** [comment]
I believe the items you need are as follows (setting in brackets to get result in picture below):

ThemedDialog - TagBackground (FF252526)
ThemedDialog - TagBorder (FF0075CC)
ThemedDialog - TagText (FF0075CC)
ThemedDialog - StartWindowDescriptionText (#FF1F1F1)

Now, a word of caution, changing these might break something else; I haven't used these settings for long enough to say either way.

![image](https://user-images.githubusercontent.com/1839845/56507198-7b136580-6518-11e9-852b-a626a15099b1.png)

**Comment by mikecoop** [comment]
Yes, that fixed it, thanks!!!!!!!! I've been annoyed about it for weeks!

**Comment by jeffaust** [comment]
It seems the names must have changed.  I can't find these anywhere.  Yes, I am on the "all elements" tab.
